### PR TITLE
Add anchors for easier deep linking

### PR DESCRIPTION
**Pull request by loisaidasam**

### CODE REVIEW

**Comment by alex** [comment]
You need to update the HTML template file, if you change this file it'll just get overritten immediately.

**Comment by alex** [comment]
Or this might be in the code, not the HTML template.

**Comment by loisaidasam** [comment]
Doh

File: print-battleground-state-changes [hunk 1]
```diff
@@ -265,7 +265,9 @@ for (state, timestamped_results) in summarized.items():
     print(f'\n{state}:')
     print(tabulate([string_summary(summary) for summary in timestamped_results]))
 
-    html_chunks.append("<table class='table table-bordered'>")
+    # TODO: Use a better slug for "North Carolina (15)" (will just be "north" for now)
```
Comment (alex) [inline print-battleground-state-changes]: If we just do `split("(")` that should work. Then replace the " " with a dash or something

Comment (loisaidasam) [reply print-battleground-state-changes]: Good point, on it ...

**Comment by loisaidasam** [comment]
```python
In [4]: 'Alaska (3)'.split('(')[0].strip().replace(' ', '-').lower()                                                                                         
Out[4]: 'alaska'

In [5]: 'North Carolina (15)'.split('(')[0].strip().replace(' ', '-').lower()                                                                                
Out[5]: 'north-carolina'
```

**Comment by alex** [comment]
Thanks!

**Comment by loisaidasam** [comment]
Thank you 🙏

**Pull request merged**